zndraw/server/tasks.py

```diff
@@ -689,3 +695,27 @@ def download_file(url: str, token: str, sid: str):
         disconnect=True,
     )
     vis.socket.emit("celery:task:emit", asdict(msg))
+
+
+@shared_task(soft_time_limit=60, time_limit=120)
+def remove_empty_rooms():
```

Comment:
I guess we'll find out if all of our python clients are properly disconnecting now 😅

Reply:
From local testing it looks promising